scripts/commands/checkOrphanPages.ts

```diff
@@ -0,0 +1,170 @@
+// This code is a Qiskit project.
+//
+// (C) Copyright IBM 2024.
+//
+// This code is licensed under the Apache License, Version 2.0. You may
+// obtain a copy of this license in the LICENSE file in the root directory
+// of this source tree or at http://www.apache.org/licenses/LICENSE-2.0.
+//
+// Any modifications or derivative works of this code must retain this
+// copyright notice, and modified files need to carry a notice indicating
+// that they have been altered from the originals.
+
+import fs from "fs/promises";
+
+import { globby } from "globby";
+import yargs from "yargs/yargs";
+import { hideBin } from "yargs/helpers";
+
+interface Arguments {
+  [x: string]: unknown;
+  currentApis: boolean;
+  devApis: boolean;
+  historicalApis: boolean;
+}
+
+// The steps to this script are:
+//  ✅ 1. Enter each folder within docs
+//  ✅ 2. Create a list of files in the _toc
+//  ✅ 3. Collect the list of files in directory
+//  ✅ 4. Check if each file in directory is in _toc
+
+// Needed features:
+//  ✅ - Check api docs if desired
+//   - Include command in .github/workflows/main.yml
```

Comment:
Leftover? It's usually better to have self-documenting code that describes the script flow, meaning that your `main` function is easy to read thanks to helper functions and friends.